components/lib/toast/ToastMessage.vue

```diff
@@ -6,7 +6,12 @@
                 <component :is="templates.messageicon ? templates.messageicon : templates.icon ? templates.icon : iconComponent && iconComponent.name ? iconComponent : 'span'" :class="cx('messageIcon')" v-bind="ptm('messageIcon')" />
                 <div :class="cx('messageText')" v-bind="ptm('messageText')">
                     <span :class="cx('summary')" v-bind="ptm('summary')">{{ message.summary }}</span>
-                    <div :class="cx('detail')" v-bind="ptm('detail')">{{ message.detail }}</div>
+                    <div :class="cx('detail')" v-bind="ptm('detail')">
+                        <template v-if="!message.detail?.__v_isVNode">
```

Comment:
I realize under the hood vue just checks `__v_isVNode` but it would it make more sense to use a supported api `isVNode` https://vuejs.org/api/render-function#isvnode?

Reply:
oh I didn't know there was helper like that. it makes much more sense. i will change it as soon as possible. thanks.